qa/tasks/thrasher.py

```diff
@@ -14,3 +15,7 @@ def exception(self):
     @exception.setter
     def exception(self, e):
         self._exception = e
+
+    @abstractmethod
+    def _do_thrash(self):
```

Comment:
What's the advantage of making this an abstract method?  I don't think it's part of the Thrasher external interface, and this class doesn't actually call it.

Reply:
I'm not sure if the `_` prefix would make it private/internal interface in python2. 

However, I didn't mean it to be a public interface here. I meant a Template Pattern here, so that the thrashers should have different implementations of `_do_thrash`.